src/main/webapp/index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Black Student Insight</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+    <script src="script.js"></script>
+  </head>
+  <body onload="hideElements()">
+    <header>
+      <nav>
+        <ul>
+          <li><a href="index.html">Black Student Insight</a></li> 
+          <li><a href="college-info.html">Find a School</a></li>
+        <!-- <li><a href="contact-us.html">Contact Us</a><li>  -->
+          <!-- future contact page? -->
+          </ul>
+        <input type="search" id="school-search" onsearch="loadSchoolInfo()" 
+        placeholder="Find a School" onkeypress="hideElements()"/>
+        <!-- <home-button onsearch="loadSchoolInfo()">Search</home-button> -->
+      </nav> 
+    </header>
```

Comment:
Take a look at the headers for the other pages and see if you can reference the formatting for this page's header.